src/Stryker.Core/Stryker.Core/TestRunners/vstest/RunEventHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Logging;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Stryker.Core.TestRunners.VsTest
```

Comment:
The namespace is called: `VsTest` but the dir is called `vstest`. I would like to see this consistent.